### PR TITLE
Create CONTRIBUTING.md

### LINKED ISSUE #56: Add docs for contributors.
Currently, there are no best practices for how to test the stack, best practices for contribution, etc. etc.

### PR DESCRIPTION
**Pull request by Victoriapm**
Closes #56 

I've added the basic guidelines on how to contribute trough PR's, work on existing issues or new ones. 
Let me know what you think!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## General Instructions
+Pull requests are the easiest and preferred method to contribute to any repo at Github.
+
+1. Fork this repository, or update if you've already forked it before. 
+2. Create a new branch specific to the issue you are working on with ``git checkout -b feature_name``. This action will separate the new feature from any other changes you may do and will make it easier to edit or ammend (read more [here](https://guides.github.com/introduction/flow/)). 
+3. Add your files on the new branch
+4. Commit locally and push to the new branch with ``git push -u origin feature_name``
+5. Go to GIthub and make a pull request, please include all the information possible.
```
Comment (biomunky) [inline CONTRIBUTING.md:13]: GIthub -> Github

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# How to contribute
+This projects welcomes contributions and corrections. Before contributing, please make sure you have read the guidelines below. 
+If you decide to contribute anything, please do the following:
+
+
+## General Instructions
```
Comment (rsyi) [inline CONTRIBUTING.md:6]: nit: can you un-capitalize "Instructions" (let's use a consistent format for all the titles)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Pull requests are the easiest and preferred method to contribute to any repo at Github.
+
+1. Fork this repository, or update if you've already forked it before. 
+2. Create a new branch specific to the issue you are working on with ``git checkout -b feature_name``. This action will separate the new feature from any other changes you may do and will make it easier to edit or ammend (read more [here](https://guides.github.com/introduction/flow/)). 
+3. Add your files on the new branch
```
Comment (rsyi) [inline CONTRIBUTING.md:11]: nit: period at the end of the sentence

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+4. Commit locally and push to the new branch with ``git push -u origin feature_name``
```
Comment (rsyi) [inline CONTRIBUTING.md:12]: nit: period

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+5. Go to GIthub and make a pull request, please include all the information possible. 
+
+## Working on Existent Issues
```
Comment (rsyi) [inline CONTRIBUTING.md:15]: same here: "Working on existent issues"

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+Please check the issues tab to find something to contribute with. Before starting to work on the selected issue make sure to check the comment thread to check that no one else has already started to work on it.
+If no one is working on it then make a comment that you have the intention to do so. This way duplicated work can be avoided. 
+
+If someone has already commented about their intentions to fix the issue but nothing has been submitted after 2 or 3 weeks, then it's acceptable to still pick up the issue but make sure to leave a comment.
+
+## Working on New Issues
```
Comment (rsyi) [inline CONTRIBUTING.md:22]: same here: "Working on new issues"

**Review by rsyi — CHANGES_REQUESTED** [review]
Thanks for taking a stab at this, @Victoriapm! This looks great. Added a few nits (and biomunky added one as well), but will merge once these are changed. :)

**Comment by rsyi** [comment]
Hey @Victoriapm -- actually going to just go ahead and merge this. Don't worry about the nits. Have some other edits to make, so I'll make them :)

**Review by rsyi — APPROVED** [review]

**Comment by Victoriapm** [comment]
Great, thanks! 
Sorry for the delay, was planning to do it tonight.

**Pull request merged**